README.md

```diff
@@ -66,6 +73,35 @@ Composable components make it easy to form a required module, but sometimes it's
 * `$ember-content-placeholders-line-height` - single line height
 * `$ember-content-placeholders-spacing` - distance between lines
 
-## 🔓 License
```

Comment:
Looks like this icon went missing in the upgrade

Reply:
Apologies, that should be fixed now! For future reference, I think project maintainers have access to edit PRs now when minor changes like this are needed. Sorry for the delay in getting back to this.